test/unit/test_runner.py

```diff
@@ -93,16 +101,13 @@ def test__execute_model__view(self,
                                   mock_adapter_rename,
                                   mock_adapter_execute_model):
         model = self.model.copy()
-
-        dbt.runner.execute_model(
-            self.profile,
-            model,
-            existing=self.existing)
+        runner = self.get_model_runner(model)
+        runner.execute(model, existing=self.existing)
 
         dbt.adapters.postgres.PostgresAdapter.drop.assert_not_called()
 
         mock_adapter_truncate.assert_not_called()
-        mock_adapter_rename.assert_not_called()
+        mock_adapter_rename.assert_called_once()
```

Comment:
i'm really dissatisfied with our current drop/create/rename logic. It works ok, but that `__dbt_tmp` suffix makes a lot of coordination between dbt components unduly difficult. I think this was an attempt at simplifying how it works, but let me revisit. Good catch